umbraco/Umbraco-CMS
# Update noUiSlider to v15.2.0

### Prerequisites


### Description
This PR upgraded noUiSlider to latest ~~v15.1.1~~ v15.2.0
https://github.com/leongersen/noUiSlider/releases

This also includes a new `drag` event, which allow to drag the connect between two handles and using the `connect` option: https://refreshless.com/nouislider/slider-options/#section-connect

```
vm.sliderOptions = {
    "start": [25, 75],
    "step": 1,
    "behaviour": 'drag-tap',
    "connect": [false, true, false],
    "tooltips": [true, true],
    "format": {
        to: function (value) {
            return value.toFixed(2);
        },
        from: function (value) {
            return Number(value);
        }
    },
    "range": {
        "min": 1,
        "max": 100
    }
};
```

![GosW4pfKOK](https://user-images.githubusercontent.com/2919859/119404858-b3319780-bce0-11eb-885b-901a41ddb327.gif)

### Test notes
- Check slider in Image Cropper still works.
- Check slider in Media Picker v3 works as expected.
- Check custom slider in e.g. a dashboard. An example here: [MyDashboard.zip](https://github.com/umbraco/Umbraco-CMS/files/6534866/MyDashboard.zip) and the implemented `onDrag` event is triggered.
